From a74b888e41592c8a71eab9b1a318af42f8345f49 Mon Sep 17 00:00:00 2001
From: Louis-Alexis Eyraud <louisalexis.eyraud@collabora.com>
Date: Thu, 10 Apr 2025 13:01:13 +0200
Subject: [PATCH 51/79] arm64: dts: mt8365-evk: Fix wrong comment in dpi0 node

On the Mediatek Genio 350-EVK board, DPI and Ethernet MII/RMII
interfaces share some GPIO pins and the SW2101 DIP switch allows
selecting whether the HDMI output or the Ethernet functionality.

Currently in the dpi0 node, there is a comment saying that the SW2101
switch needs to be on 'LAN' position to make HDMI work, so fix the
comment to say 'HDMI' instead

Fixes: 2c3df90c2072 ('arm64: dts: mediatek: add ethernet support for mt8365-evk')
Signed-off-by: Louis-Alexis Eyraud <louisalexis.eyraud@collabora.com>
---
 arch/arm64/boot/dts/mediatek/mt8365-evk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/mediatek/mt8365-evk.dts b/arch/arm64/boot/dts/mediatek/mt8365-evk.dts
index bd7bf23da183..d8996daf0f4c 100644
--- a/arch/arm64/boot/dts/mediatek/mt8365-evk.dts
+++ b/arch/arm64/boot/dts/mediatek/mt8365-evk.dts
@@ -183,7 +183,7 @@ &dpi0 {
 	/*
 	 * Ethernet and HDMI (DPI0) are sharing pins.
 	 * Only one can be enabled at a time and require the physical switch
-	 * SW2101 to be set on LAN position
+	 * SW2101 to be set on HDMI position
 	 */
 	status = "disabled";
 
-- 
2.43.0

